modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java

```diff
@@ -376,7 +393,7 @@ private String _getOptionReferenceValue(
 		throws JSONException, ParseException {
 
 		DDMFormField ddmFormField = ddmFormFieldsMap.get(
-			ddmFormFieldValue.getName());
```

Comment:
Hey @p-albuquerque,
Why it was relevant to change the way of getting the ddmFormFieldsMap from the property fieldReference?